### ISSUE TITLE
Create a chatbot with coral and raspberry pi 4b

### ISSUE THREAD

**Issue by LoKoNaoX** [google-coral/pycoral#139]
### Description

I'm trying to create a chatbot with coral and raspberry pi. But I have used LLama2 models to create the text recognosation and I can make question to the model about the text and documents. It is posible to do that with coral o use it to proccess all the information?

<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Support, Feature Request

### Operating System

Linux

### Coral Device

USB Accelerator

### Other Devices

Rapsberry Pi 4

### Relevant Log Output

_No response_</details>

**Comment by nvtkaszpir** [comment]
no, coral has not enough memory, everything would be on the host and not on the device.
